package.json

```diff
@@ -42,5 +42,8 @@
     "uglify-js": "^3.17.4",
     "webpack": "^5.72.1",
     "webpack-cli": "^4.9.2"
+  },
+  "dependencies": {
+    "markdown-it": "^13.0.1"
```

Comment:
@gracexu94, Sorry, meant to say should all of these devDependencies live in the dependencies as well?

Reply:
Yes, and let's update the references to the starter template to be the DSC repo/site.

Reply:
Resolved in https://github.com/kindsys/digital-services-coalition/pull/23/commits/1d972e903ec4ee6a82b74802e85ea1e1387d5db3. Let me know if the homepage link is correct, as it originally had it linked to the template repo's readme